packer/scripts/server-setup.ps1

```diff
@@ -171,10 +179,25 @@ function Install-PreRequisites() {
 
     Install-Module -Name SqlServer -MinimumVersion '21.1.18068' -Scope CurrentUser -Force -AllowClobber
 
+    RemoveDesktopIcons
+    
     Stop-Transcript
 }
 
+function Set-WallPaper {
+    Write-Output "Downloading wallpaper image"
+    $url = "https://edfidata.s3-us-west-2.amazonaws.com/Starter+Kits/images/EdFiQuickStartBackground.png"
+    $downloadedFile = Get-FileFromInternet $url
+	
+    Copy-Item -Path $downloadedFile -Destination "C:/"
```

Comment:
@raghuramakrishna copy it under user directory or under C:\Ed-Fi-Starter-Kit

Reply:
updated